themes/default/content/docs/concepts/inputs-outputs/_index.md

```diff
@@ -522,15 +522,18 @@ connection_string = Output.all(server=sql_server.name, db=database.name) \
 {{% choosable language go %}}
 
 ```go
+
 connectionString := pulumi.All(sqlServer.Name, database.Name).ApplyT(
-    func (args []interface{}) (string, error) {
-        server := args[0].(string)
-        db := args[1].(string)
-        return fmt.Sprintf("Server=tcp:%s.database.windows.net;initial catalog=%s...", server, db), nil
-    },
+    func (args []interface{}) pulumi.StringOutput  {
+        server := args[0]
+        db := args[1]
+        return pulumi.Sprintf("Server=tcp:%s.database.windows.net;initial catalog=%s...", server, db)
+    }
 )
 ```
 
+**A note on error handling** The function `ApplyT` spawns a Goroutine to await the availability of the implicated dependencies. This function accepts a `T` or `(T, error)` signature; the latter accomodates for error handling. Alternatively, one may use the `ApplyTWithContext` function in which the provided context can be used to reject the output as canceled. Error handling may also be achieved using an `error` `chan`.
+
```

Comment:
If you were thinking of this as a callout, we have these which you might want to use instead:

```suggestion
{{% notes %}}
**A note on error handling** The function `ApplyT` spawns a Goroutine to await the availability of the implicated dependencies. This function accepts a `T` or `(T, error)` signature; the latter accomodates for error handling. Alternatively, one may use the `ApplyTWithContext` function in which the provided context can be used to reject the output as canceled. Error handling may also be achieved using an `error` `chan`.
{{% /notes %}}
```

They look like this:

![image](https://github.com/pulumi/pulumi-hugo/assets/274700/2afb7f90-4b4a-4e6c-b189-4801e18c2912)